build-utils/Dockerfile-BuildDeploy

```diff
@@ -114,6 +114,8 @@ RUN apk  --no-cache add \
     ruby \
     ruby-bundler \
     ruby-json  \
+    make \
+    g++ \
     && rm -rf /var/cache/apk/*
```

Comment:
npm install sometimes wants to build some binaries. These are the programs it wants to use to do so.